File: Javascript/A_NivelIniciante/Exercicio_1044.js
var input = require("./readFile.js").input;
var lines = input.split('\n');
var line = lines[0].split(" ")
var a = line[0]*1
var b = line[1]*1
console.log(a>b? a%b==0?"Sao Multiplos" : "Nao sao Multiplos" : b%a==0?"Sao Multiplos":"Nao sao Multiplos" )

File: Python/A_NivelIniciante/Exercicio_1044.py
# -*- coding: cp1252 -*-
# -*- encoding: utf-8 -*-
#url do problema - https://www.urionlinejudge.com.br/judge/pt/problems/view/1044
linha1 = raw_input()
A,B = linha1.split(" ")
A = int(A)
B = int(B)
if A%B==0 or B%A==0:
    print("Sao Multiplos")
else:
    print("Nao sao Multiplos")
   
